spec/linter-eslint-spec.js

```diff
@@ -1,12 +1,17 @@
 'use babel'
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
To be honest, it would probably be better to make this change in a config file for the whole specs directory.  I need to run out now, but I can get to it later if you want to block the merge.  Otherwise, I'll make the change as part of a subsequent effort to get #666 passing.